serverlessworkflow/modules/ROOT/pages/eventing/consume-produce-events-with-knative-eventing.adoc

```diff
@@ -9,178 +9,184 @@
-|Broker
-|Kubernetes Kind of the generated Knative Sink.
+|`org.kie.kogito.addons.knative.sink.kind`
+|`Broker`
+|This property indicates the Kubernetes kind of the generated Knative sink.
 
 |===
+--
+
+[[ref-knative-eventing-add-on-sink-configuration]]
+== Knative Eventing add-on sink configuration
 
-[[con-knative-eventing-add-on-sink-configuration]]
-== Knative Eventing add-on Sink configuration
+The configuration described in this section is useful if your Serverless Workflow consists of at least one `consumed` type link:{spec_doc_url}#Event-Definition[event definition]. In this scenario, the workflow application consumes events, acting as a Knative sink.
 
-The description in this section is only valid if the workflow has at least one event definition of the type `consumed`. That's when the workflow application consume events, acting as a Knative Sink.
+When the workflow application needs to consume events, the Knative Eventing add-on generates link:{knative_eventing_doc_url}/broker/triggers[Knative triggers]. The Knative triggers are configured to listen to a broker with the required event type, which is defined in your Serverless Workflow definition.
 
-When the workflow application needs to consume events, the Knative Eventing add-on generates link:{knative_eventing_doc_url}/broker/triggers[Knative Triggers] configured specifically to listen from a Broker with the required event type as defined in your workflow definition.
+The following table lists the configuration property related to Knative sink generation:
 
-.Knative Sink generation configuration
+.Knative sink generation configuration property
 |===
-|Property|Default|Description
+|Property|Default value|Description
 
-|org.kie.kogito.addons.knative.broker
+|`org.kie.kogito.addons.knative.broker`
 |`default`
-|Name of the default Knative Broker deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative Triggers responsible to delegate the events that this Kogito service will consume.
+|This property indicates the name of the default Knative broker that is deployed in the Kubernetes namespace. This broker is used as the reference to create the Knative triggers, which are responsible to delegate the events that the Kogito service consumes.
 
 |===
 
-[[con-generating-kn-objects-build-time]]
-== Generating Knative Objects in Build Time
+[[proc-generating-kn-objects-build-time]]
+== Generating Knative objects during build time
 
-Kogito can generate Knative objects during the workflow application build time to facilitate the deployment in a Kubernetes cluster.
-
-[NOTE]
-====
-You can ignore this section if you plan to configure and deploy the Knative objects yourself.
-====
+Kogito can generate Knative objects during the workflow application build time to facilitate the deployment in a Kubernetes cluster. The procedure described in this section is useful if you plan to configure and deploy the Knative objects.
```

Comment:
```suggestion
Kogito can generate Knative objects during the workflow application build time to facilitate the deployment in a Kubernetes cluster. However, you don't need to use these objects if you plan to create and deploy them by yourself.
```

Reply:
This section should suggest that the objects generated by Kogito are optional and a suggestive approach to deploying the application on Knative. Users can opt to create these objects manually or use any other tool/method.

Reply:
Got it, thanks
Changed the sentence as follows:

However, you do not need to use the generated objects if you plan to create and deploy the Knative objects by yourself.